services/streammanager/kafka/kafkamanager.go

```diff
@@ -0,0 +1,178 @@
+package kafka
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/rudderlabs/rudder-server/config"
+	"github.com/rudderlabs/rudder-server/utils/logger"
+	"github.com/tidwall/gjson"
+)
+
+// Config is the config that is required to send data to Kafka
+type Config struct {
+	Topic         string
+	HostName      string
+	Port          string
+	SslEnabled    bool
+	CACertificate string
+}
+
+var (
+	clientCertFile, clientKeyFile string
+	certificate                   tls.Certificate
+)
+
+func init() {
+	loadConfig()
+	loadCertificate()
+}
+
+func loadConfig() {
+	clientCertFile = config.GetEnv("SSL_CERTIFICATE_FILE_PATH", "")
+	clientKeyFile = config.GetEnv("SSL_KEY_FILE_PATH", "")
+}
+
+func loadCertificate() {
+	if clientCertFile != "" && clientKeyFile != "" {
+		cert, err := tls.LoadX509KeyPair(clientCertFile, clientKeyFile)
+		if err != nil {
+			logger.Errorf("error loading certificate: ", err)
```

Comment:
Let's panic here. There should not be an invalid file.

Reply:
Incorporated.